<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编码转换工具V1.0.0(2020.03.16)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>当前路径:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPath">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QGroupBox" name="groupBoxPath">
          <property name="title">
           <string>路径设置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>选择文件</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_2">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QPushButton" name="btnChooseFile">
                    <property name="text">
                     <string>选择文件...</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_2">
              <attribute name="title">
               <string>选择文件夹</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_3">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QPushButton" name="btnChooseFolder">
                      <property name="text">
                       <string>选择文件夹...</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                     <string>递归处理文件夹中以下类型</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_3">
                     <item row="1" column="1">
                      <widget class="QCheckBox" name="cbTxtFile">
                       <property name="text">
                        <string>.txt</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="3">
                      <widget class="QCheckBox" name="cbJavaFile">
                       <property name="text">
                        <string>.java</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="2">
                      <widget class="QCheckBox" name="cbCppFile">
                       <property name="text">
                        <string>.cpp</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item row="8" column="0" colspan="4">
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>如果自定义多种文件类型，不同类型之间用空格分开，如：.html .xml</string>
                       </property>
                       <property name="wordWrap">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QCheckBox" name="cbCFile">
                       <property name="text">
                        <string>.c</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QCheckBox" name="cbBatFile">
                       <property name="text">
                        <string>.bat</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="2">
                      <widget class="QCheckBox" name="cbPyFile">
                       <property name="text">
                        <string>.py</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QCheckBox" name="cbMFile">
                       <property name="text">
                        <string>.m</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QCheckBox" name="cbHFile">
                       <property name="text">
                        <string>.h</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="3">
                      <widget class="QCheckBox" name="cbHppFile">
                       <property name="text">
                        <string>.hpp</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item row="6" column="0" colspan="2">
                      <widget class="QLineEdit" name="leditCustomEncode"/>
                     </item>
                     <item row="6" column="2" colspan="2">
                      <widget class="QPushButton" name="btnCustomCheck">
                       <property name="text">
                        <string>应用</string>
                       </property>
                      </widget>
                     </item>
                     <item row="5" column="0" colspan="4">
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>自定义类型</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxEncode">
          <property name="title">
           <string>编码设置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string>转换后的编码:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>UTF-8 BOM</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>UTF-8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GB2312</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="btnTransmit">
            <property name="text">
             <string>转换</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClear">
            <property name="text">
             <string>清屏</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
